kombu/transport/pykafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.pykafka
```

Comment:
@Dowwie no, I don't. I guess this is because of the file rename :/ . I still have the comments cached on other browser tab. Should I paste the whole thing here?